README.md

```diff
@@ -72,6 +79,75 @@ resource "aws_security_group_rule" "container_instance_https_egress" {
 }
 ```
 
+### Auto Scaling
+
+This module creates an Auto Scaling group for the ECS cluster. By default, there are no Auto Scaling policies associated with this group. In order for Auto Scaling to function, you must define `aws_autoscaling_policy` resources and associate them with the `container_instance_autoscaling_group_name` module output.
+
+See this [article](https://segment.com/blog/when-aws-autoscale-doesn-t/) for more information on Auto Scaling, and below for example policies.
+
+```hcl
+resource "aws_autoscaling_policy" "container_instance_cpu_reservation" {
+  name                   = "asgScalingPolicyCPUReservation"
+  autoscaling_group_name = "${module.container_service_cluster.container_instance_autoscaling_group_name}"
+  adjustment_type        = "ChangeInCapacity"
+  policy_type            = "TargetTrackingScaling"
+
+  target_tracking_configuration {
+    customized_metric_specification {
+      metric_dimension {
+        name  = "ClusterName"
+        value = "${module.container_service_cluster.name}"
+      }
+
+      metric_name = "CPUReservation"
+      namespace   = "AWS/ECS"
+      statistic   = "Average"
+    }
+
+    target_value = "50.0"
+  }
+}
+
+resource "aws_autoscaling_policy" "container_instance_memory_reservation" {
+  name                   = "asgScalingPolicyMemoryReservation"
+  autoscaling_group_name = "${module.container_service_cluster.container_instance_autoscaling_group_name}"
+  adjustment_type        = "ChangeInCapacity"
+  policy_type            = "TargetTrackingScaling"
+
+  target_tracking_configuration {
+    customized_metric_specification {
+      metric_dimension {
+        name  = "ClusterName"
+        value = "${module.container_service_cluster.name}"
+      }
+
+      metric_name = "MemoryReservation"
+      namespace   = "AWS/ECS"
+      statistic   = "Average"
+    }
+
+    target_value = "90.0"
+  }
+}
+```
+
+It's worth nothing that the [`aws_autoscaling_policy`](https://www.terraform.io/docs/providers/aws/r/autoscaling_policy.html) documentation suggests we remove `desired_capacity` from the `aws_autoscaling_group` resource when using Auto Scaling. That makes sense, because when it is present, any Terraform plan/apply cycle will reset it.
```

Comment:
```suggestion
It's worth noting that the [`aws_autoscaling_policy`](https://www.terraform.io/docs/providers/aws/r/autoscaling_policy.html) documentation suggests `desired_capacity` be removed from the `aws_autoscaling_group` resource when using Auto Scaling. Since `desired_capacity` exists in this module, users will have to dynamically override the value passed on each Terraform plan/apply cycle.
```

Reply:
Good catch.